buildSrc/src/main/groovy/csde.gradle

```diff
@@ -113,6 +114,9 @@ def projectContext = new ProjectContext(project,
         cordaNotaryPluginsVersion
 )
 
+// todo: redirect the config file
```

Comment:
I don't follow the TODO comment - WDYM 'redirect' ?

Reply:
Removed, I'd done it already.